plugins/package-managers/bazel/src/main/kotlin/MultiBazelModuleRegistryService.kt

```diff
@@ -50,6 +52,11 @@ internal class MultiBazelModuleRegistryService(
             // Add the default Bazel registry as a fallback.
             val registryUrls = (urls + DEFAULT_URL).distinctBy { it.removeSuffix("/") }
 
+            logger.info { "Creating a multi Bazel module registry:" }
```

Comment:
IMO this should be `debug` instead of `info`.

Reply:
Since the logging occurs only one, I think there is no harm in letting it to `info`, especially considering how noisy `debug` is.